File: official/docs/csharp/shipments/OneCallBuyShipment.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Shipment shipment = await Shipment.Create(new Dictionary<string, object>() {
  { "carrier_accounts", new List<string>() {
      "ca_..."
  } },
  { "service", "NextDayAir"},
  { "parcel", new Dictionary<string, object>() {
    { "length", 20.2 },
    { "width", 10.9 },
    { "height", 5 },
    { "weight", 65.9 }
  } },
  { "from_address", new Dictionary<string, string>() {
    { "name", "EasyPost" },
    { "street1", "417 Montgomery Street" },
    { "street2", "5th Floor" },
    { "city", "San Francisco" },
    { "state", "CA" },
    { "zip", "94104" },
    { "country", "US" },
    { "phone", "4153334445" },
    { "email", "support@easypost.com"}
  } },
  { "to_address", new Dictionary<string, string>() {
    { "name", "Dr. Steve Brule" },
    { "street1", "179 N Harbor Dr" },
    { "city", "Redondo Beach" },
    { "state", "CA" },
    { "zip", "90277" },
    { "country", "US" },
    { "phone", "8573875756" },
    { "email", "dr_steve_brule@gmail.com" }
  } }
});

Console.WriteLine(JsonConvert.SerializeObject(shipment, Formatting.Indented));

File: official/docs/node/shipments/one_call_buy_shipment.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

const shipment = new api.Shipment({
  carrier_accounts: ['ca_...'],
  service: 'NextDayAir',
  to_address: {
    name: 'Dr. Steve Brule',
    street1: '179 N Harbor Dr',
    city: 'Redondo Beach',
    state: 'CA',
    zip: '90277',
    country: 'US',
    email: 'dr_steve_brule@gmail.com',
    phone: '4155559999',
  },
  from_address: {
    street1: '417 montgomery street',
    street2: 'FL 5',
    city: 'San Francisco',
    state: 'CA',
    zip: '94104',
    country: 'US',
    company: 'EasyPost',
    phone: '415-123-4567',
  },
  parcel: {
    length: 20.2,
    width: 10.9,
    height: 5,
    weight: 65.9,
  },
});

shipment.save().then(console.log);
